frontend/pages/smileLoginRoute/components/Footer/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import I18n from '@shopgate/pwa-common/components/I18n';
+import styles from './style';
+import getConfig from '../../../../helpers/getConfig';
+
+const { shopifyAlias } = getConfig();
+
+/**
+ * TODO: <div className={styles.icon} /> get the svg icon
+ * @returns {JSX}
+ */
+const Footer = () => (
+  <div className={styles.container}>
+    <a className={styles.text} href={`https://smile.io?utm_medium=smile-panel&utm_campaign=panel-referral&utm_source=https%3A%2F%2F${shopifyAlias}.myshopify.com`}>
```

Comment:
utm_source can't be hardcoded for shopify.
I also think it should not use the desktop page url. We should create something like {nameOfShop}-App

Reply:
I'm a bit confused about this one. Maybe we can have a call about this one.

Reply:
Changed to use shopName for utm_source